onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -128,6 +128,7 @@ namespace perftest {
       "\t\t The number of affinities must be equal to intra_op_num_threads - 1\n\n"
       "\t-D [Disable thread spinning]: disable spinning entirely for thread owned by onnxruntime intra-op thread pool.\n"
       "\t-Z [Force thread to stop spinning between runs]: disallow thread from spinning during runs to reduce cpu usage.\n"
+      "\t-n [Exit after session creation]: allow user to dump optimized model or EP context cache model.\n"
```

Comment:
need more work in that case.

---
In reply to: [1486863469](https://github.com/microsoft/onnxruntime/pull/19501#discussion_r1486863469) [](ancestors = 1486863469)

Reply:
ok. yeah, let's not rush to merge this one then. i think it should still measure the session creation time. 
otherwise, i don't think exit after session creation should be an option in perf_test tool.

Reply:
done

---
In reply to: [1486893316](https://github.com/microsoft/onnxruntime/pull/19501#discussion_r1486893316) [](ancestors = 1486893316)